app/controllers/api/v1/admin/base_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::Admin::BaseController < Api::V1::ApiController
+
+  def require_admin
+    current_user && current_user.has_role?("admin") || doorkeeper_authorize!
```

Comment:
To your note in the PR message, I think we'll eventaully want to change `doorkeeper_authorize!` to do something similar but check the app scopes first (to require admin or whatever we call it). Am I right, we'll do that work as part of https://trello.com/c/z2gIuTBs/279-default-oauth-app-scopes-to-sandbox?